browser/ephemeral_storage/sources.gni

```diff
@@ -0,0 +1,14 @@
+brave_browser_ephemeral_storage_sources = [
+  "//brave/browser/ephemeral_storage/ephemeral_storage_tab_helper.cc",
+  "//brave/browser/ephemeral_storage/ephemeral_storage_tab_helper.h",
+]
+
+brave_browser_ephemeral_storage_deps = [
+  "//base",
+  "//chrome/browser/profiles",
```

Comment:
AFAIK it doesn't add a circular dep, so these sources/deps can stay in build.gn

Reply:
sadly it adds.